README.md

```diff
@@ -25,16 +26,14 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 <button data-ng-click="vm.create()">Click</button>
 ```
 
-```text
-Your answer goes here...
+```ng-click only allows you to specify a behavior when you click on an element, data-ng-click will pass data to the method when you click the object.
```

Comment:
Adding `data-` to the beginning of a directive allows our HTML to validate https://validator.w3.org/